routes/api/quotes.js

```diff
@@ -0,0 +1,22 @@
+const express = require("express");
+const router = express.Router();
+
+//load the quotes JSON
+const quotes = require("../../quotes.json");
+const quotesWithId = require("../../quotes-with-id.json");
+
+router.get("/", (req, res) => res.json(quotes));
+
+router.get("/random", (req, res) => {
+  res.send(pickFromArray(quotesWithId));
+});
+
+router.get("/:id", (req, res) => {
+  res.send(quotesWithId.find((quote) => quote.id === Number(req.params.id)));
```

Comment:
This is quite verbose. Consider this alternative:

```suggestion
  const id = Number(req.params.id)
  const quote = quotesWithId.find((quote) => quote.id === id)
  res.send(quote);
```

I'm not saying one is better than the other, but it's worth considering which 'philosophy' you prefer here, and why.

Reply:
That's better